src/set_theory/cardinal/cofinality.lean

```diff
@@ -705,14 +704,61 @@ local infixr ^ := @pow cardinal.{u} cardinal cardinal.has_pow
 def is_limit (c : cardinal) : Prop :=
 c ≠ 0 ∧ ∀ x < c, succ x < c
 
+theorem is_limit.ne_zero {c} (h : is_limit c) : c ≠ 0 :=
+h.1
+
+theorem is_limit.succ_lt {x c} (h : is_limit c) (hx : x < c) : succ x < c :=
+h.2 x hx
+
+theorem is_limit.omega_le {c} (h : is_limit c) : ω ≤ c :=
+begin
+  by_contra' h',
+  rcases lt_omega.1 h' with ⟨_ | n, rfl⟩,
+  { exact h.1.irrefl },
+  { simpa using h.2 n }
+end
+
 /-- A cardinal is a strong limit if it is not zero and it is
   closed under powersets. Note that `ω` is a strong limit by this definition. -/
 def is_strong_limit (c : cardinal) : Prop :=
 c ≠ 0 ∧ ∀ x < c, 2 ^ x < c
 
+theorem is_strong_limit.ne_zero {c} (h : is_strong_limit c) : c ≠ 0 :=
+h.1
+
+theorem is_strong_limit.power_lt {x c} (h : is_strong_limit c) (hx : x < c) : 2 ^ x < c :=
+h.2 x hx
+
 theorem is_strong_limit.is_limit {c} (H : is_strong_limit c) : is_limit c :=
 ⟨H.1, λ x h, lt_of_le_of_lt (succ_le_of_lt $ cantor x) (H.2 _ h)⟩
 
+theorem mk_subset_mk_lt_cof {α : Type*} (h : is_strong_limit (#α)) :
+  #{s : set α // #s < cof (#α).ord} = #α :=
```

Comment:
This is a rather Con(NF)-specific statement. I'm very happy to have it a separate lemma, but it should be deduced from the version about bounded sets.